### ISSUE TITLE
Rename repo?

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js-pwa#474]
@kelson42 Some advice, please.

**TLDR** - Would you be in favour of my renaming this repo to _**kiwix-js-pwa**_?

***Rationale / discussion:***

As you know, this repo started as a place to customize kiwix-js for a Windows UWP app. However, for the last four to five years, the UWP app has no longer been the main target. Due to decisions taken by MS, the UWP technology is no longer the "Universal" solution it was originally touted as being, and has been largely deprecated in favour of PWAs and Electron-based apps (also WebView2 apps, which are similar to Electron, but I don't target that type).

Both PWA and Electron are of course much more "universal" than UWP (a Windows-only technology) ever was. And it's become clear to me over the last two years, that the future of this app is largely PWA (with Electron just being one way of packaging a PWA for offline installation).

So, I wonder if it is now time to rename kiwix-js-windows to kiwix-js-pwa?

Now the only potential complication is that kiwix-js upstream (the browser extension) is also a PWA for both Chromium extensions and for Firefox extensions, simply because PWA is the only solution allowing inline JS to work in an extension. Nevertheless, I think there is a conceptual difference between both repositories, given that the **main** targets of kiwix-js are the desktop browser extensions, whereas the **main** target of this repositroy has now become a specialized PWA using latest PWA technologies (with graceful fallbacks all the way to IE11).

NB I don't think renaming this Repo precludes unifying the two apps in the future at some point, or backporting stable technologies into kiwix-js until there is less difference between the two. It might also be worth noting that just over a year ago a user started an issue saying the current title is misleading: #296.

**Comment by kelson42** [comment]
LGTM

**Comment by rgaudin** [comment]
Nope, this line regards the name of the github repository that runs the workflow. It's just a safety net to avoid people forking the repo to accidentally run the workflow. Nothing to do on registry side as you're keeping the same image name

**Comment by Jaifroid** [comment]
Great, thanks @rgaudin!
